GeoQuiz/app/src/main/java/com/bignerdranch/android/geoquiz/CheatActivity.kt

```diff
@@ -0,0 +1,75 @@
+package com.bignerdranch.android.geoquiz
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Button
+import android.widget.TextView
+
+private const val TAG = "CheatActivity"
+const val EXTRA_ANSWER_SHOWN = "com.bignerdbranch.android.geoquiz.answer_shown"
```

Comment:
if this can't be private, you can group values by their access levels: i.e group up private, leave an empty line, group those with default access (even if it's only one now)